src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from "react";
+import { Todo } from "../../types/Todo";
+import cn from 'classnames';
+
+interface Props {
+  todos: Todo[];
+  setFilteredTodos: (b: Todo[]) => void;
+  setTodos: (b: Todo[]) => void;
+}
+
+export const Footer = ({ todos, setFilteredTodos, setTodos }: Props) => {
+  const [filtered, setFiltered] = useState('all');
```

Comment:
move all filtered values to some enum, and reuse everywhere

Reply:
move this state to App.tsx and pass to Footer the filtered and setFiltered props